client/src/components/InlinePanel/index.test.js

```diff
@@ -10,63 +10,77 @@ describe('InlinePanel', () => {
   const handleReadyEvent = jest.fn();
 
   const onAdd = jest.fn();
+  let panel;
 
   beforeAll(() => {
     $.fx.off = true;
     jest.resetAllMocks();
 
+    const prefix = 'person_cafe_relationship';
+    const childPrefix = `${prefix}-__prefix__`;
     document.body.innerHTML = `
   <form>
-    <input name="person_cafe_relationship-TOTAL_FORMS" value="0" id="id_person_cafe_relationship-TOTAL_FORMS" type="hidden" />
-    <input name="person_cafe_relationship-INITIAL_FORMS" value="0" id="id_person_cafe_relationship-INITIAL_FORMS" type="hidden" />
-    <input name="person_cafe_relationship-MIN_NUM_FORMS" value="1" id="id_person_cafe_relationship-MIN_NUM_FORMS" type="hidden" />
-    <input name="person_cafe_relationship-MAX_NUM_FORMS" value="5" id="id_person_cafe_relationship-MAX_NUM_FORMS" type="hidden" />
-    <div id="id_person_cafe_relationship-FORMS"></div>
-    <template id="id_person_cafe_relationship-EMPTY_FORM_TEMPLATE">
-      <div id="inline_child_person_cafe_relationship-__prefix__" data-child-form-mock>
-        <p>Form for inline child</div>
```

Comment:
There is a lot more refactoring needed in this file, as the test cases are currently inter-dependent, but I didn’t want to make code review even harder.

For example see this highlighted line which had invalid markup in the test HTML.

Reply:
🥲